BUG BOUNTY.md

```diff
@@ -0,0 +1,187 @@
+# Bug Bounty at System Initiative
+
+**Launch Date:** 27th November, 2024 | **Last updated:** 27th November, 2024  
```

Comment:
Is this not implied by the git history? If we want this, that works for me! Just curious.

Reply:
In theory yes, might be worth removing the Last Updated. Launch date is still relevant

Reply:
Sorry i missed this before merging, I can come back to it and remove it